packages/jest-runtime/package.json

```diff
@@ -8,6 +8,7 @@
   "license": "MIT",
   "main": "build/index.js",
   "dependencies": {
+    "babel-core": "^6.0.0",
     "babel-jest": "^21.2.0",
```

Comment:
we don't need to decouple this, right? I haven't dug into how this works, yet...